include/dxc/Support/SPIRVOptions.h

```diff
@@ -54,17 +53,14 @@ struct EmitSPIRVOptions {
   std::vector<std::string> bindRegister;
   llvm::SmallVector<llvm::StringRef, 4> allowedExtensions;
   llvm::StringRef targetEnv;
-  spirv::LayoutRule cBufferLayoutRule;
-  spirv::LayoutRule tBufferLayoutRule;
-  spirv::LayoutRule sBufferLayoutRule;
+  SpirvLayoutRule cBufferLayoutRule;
```

Comment:
Let's also reorder these fields according to their type and name.

Reply:
Done